libs/langchain/langchain/retrievers/multi_vector.py

```diff
@@ -20,6 +30,8 @@ class MultiVectorRetriever(BaseRetriever):
     id_key: str = "doc_id"
     search_kwargs: dict = Field(default_factory=dict)
     """Keyword arguments to pass to the search function."""
+    search_type: SearchType = SearchType.similarity
```

Comment:
ooc will this raise validation error if someone passes in a string? if so would prefer to just type as 
```python
search_type: Literal["similarity", "mmr"] = "similarity"
```